Porting/vote_admin_guide.pod

```diff
@@ -0,0 +1,176 @@
+
+=head3 Start the poll
+
+You'll get an email linking back to the poll's control page.
+
+B<BOOKMARK THE CONTROL PAGE!>
+
+Don't lose that link or you won't be able to administer the poll and you'll
+have to re-create it.
+
+From here, you can double check your settings.  Some things, but not all,
+can be edited.
+
+If it looks good, click "start poll."  You'll get a success page and can
+click "go back to poll control".
+
+=head3 Check how the email looks
+
+At the bottom, under "Add voters", enter your email address.  Click "add
+voters."  You'll get another success screen and can again "go back to poll
+control."
+
+Meanwhile, you'll get another email, this one linking you to the voting
+page itself.  Proofread the email.  Click the link. Proofread the page.  If
+you want, cast your vote!
+
+At this point, you probably either need to start over - because you can't
+edit a poll once it's going - or you're good to proceed.   If you need to
+start over, just close the poll (or not, but it seems polite to do so) and
+start over.  Otherwise...
+
+=head3 Send the poll!
+
+This is the point of no return!  Once you send the poll to the whole team,
+you'll look silly if you made a big mistake.  That's okay, you can survive
+looking silly, but it's better to avoid it if you can.
+
+Put all the email addresses for the team into the "add voters" on the poll
+control page.  There should be as many of these as there are team members.
+Double check!  It's okay if you enter your email address, again.  You won't
+be allowed to vote twice.
+
+Click "add voters."  It will take a little while.  (I think it does all the
+email sending synchronously.)  When that's done, you can email the list to
+say the ballot has been sent.
+
+If you're told that some email addresses aren't authorized to vote, chase
+down these Core Team members that haven't opted-in their email address.
+
+=head3 Wait
+
+Now you just need to wait for the end date.  Set a calendar alarm to remind
+yourself.  Look at the poll control page once in a while to see how many votes
+were cast.  Because votes can't be edited once cast, you could end the voting
+immediately once 100% of ballots have been cast, but don't expect this will
+actually happen.
+
+=head3 End the poll
+
+When it's time, go to the poll control page and click "end poll".  You'll get
+dumped on a results page that will provide the three winners (and a bunch of
+other data).  Use the "minimax" rule (the default).  Everyone else will be able
+to access these results, so don't worry about copying anything but the winners.
+Inform the group!
+
+=head2 After an election
+
+In addition to notifying the Perl Core mailing list of the results, open a
+pull request to update perlgov.pod with the new Steering Council members.
+If a non-Core-Team candidate won, also add them to the Core Team list.
+
+Ask the outgoing Steering Council to change the Perl Core mailing list
+administrators to match the incoming Steering Council.
```

Comment:
```suggestion
Ask the outgoing Steering Council to:

=over

=item *

edit the Perl Core mailing list admins to match the incoming Steering Council

=item *

update the GitHub "steering" team to match incoming Steering Council

=item *
 
request that the Perl NOC update the perl-security list to include all new council members

=back
```

Reply:
Edited and included in a separate commit.